### ISSUE TITLE
[Ray Dashboard] Node link in the event doesn't work 

### ISSUE THREAD

**Issue by scottsun94** [ray-project/ray#33977]
### What happened + What you expected to happen

<img width="1013" alt="Screen Shot 2023-03-31 at 1 51 20 PM" src="https://user-images.githubusercontent.com/9677264/229227902-cbe75544-5dc2-4610-8e61-d6e9a51679a3.png">
Click the link and see an empty tab.
<img width="1334" alt="Screen Shot 2023-03-31 at 1 51 26 PM" src="https://user-images.githubusercontent.com/9677264/229227911-b009c82d-d7bb-4564-a55c-c7c002dfefa1.png">

### Versions / Dependencies

nightly

### Reproduction script

n/a

### Issue Severity

Low: It annoys or frustrates me.

**Comment by rkooo567** [comment]
Good catch. I think we should revamp this UI and allow to jump to the table for relevant ids (task id, actor id, pg id, node id).
